### ISSUE TITLE
False positive for 'aria-rowcount' and 'aria-colcount' used on role 'grid'

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#547]
-  'aria-rowcount' and 'aria-colcount' should be allowed on role ‘grid’ instead of role ‘row’, as they represent the total number if rows and columns in a grid. 

- aria-colindex should be allowed with role row since it represents the index position of the row in a table or grid.

**Comment by isner** [comment]
Thanks for pointing this out. I've addressed the issue in https://github.com/dequelabs/axe-core/pull/555.

**Comment by iamrafan** [comment]
@isner, thanks for the fix

**Comment by marcysutton** [comment]
The fix for this got merged, so I'll close this issue. It will be out in our next release!
